frontend/tests/controllers/UserDeleteTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * UserDeleteTest
+ */
+​
+class UserDeleteTest extends \OmegaUp\Test\ControllerTestCase {
+    /**
+     * Deletes an omegaup user as admin
+     */
+    public function testDeleteUserAsAdmin() {
+        $username = \OmegaUp\Test\Utils::createRandomString();
+        $email = $username . '@' . $username . '.com';
```

Comment:
```suggestion
        $email = "{$username}@{$username}.com";
```

this is a bit more succinct.

Reply:
Okay! Done.